src/main/java/nextstep/subway/applicaion/LineService.java

```diff
@@ -34,18 +34,17 @@ public LineResponse saveLine(LineRequest request) {
             Station downStation = stationService.findById(request.getDownStationId());
             line.getSections().add(new Section(line, upStation, downStation, request.getDistance()));
         }
-        return line.createLineResponse();
+        return createLineResponse(line);
     }
 
     public List<LineResponse> showLines() {
         return lineRepository.findAll().stream()
-                .map(Line::createLineResponse)
+                .map(this::createLineResponse)
```

Comment:
```suggestion
                .map(LineResponse::from)
```
이런식으로하고 Response객체에서 알아서 생성하도록 해도 되었을 것 같네요 ㅎㅎ